### ISSUE TITLE
Desafio 01 - móduo 1 

### ISSUE THREAD

**Issue by isadhoramauricio** [gustavoguanabara/html-css#167]
dá pra mudar o espaçamento entre linhas?? meu código ficou com o espaçamento bem maior que do professor guanabara, n sei se foi o código que escrevi errado.... ou uma configuração no Visual Studio Code que não fiz.... 

gostaria de ajuda nisso

**Comment by Djh0w** [comment]
Sim, da para mudar o espaçamento, mas via estilização CSS. No seu caso, pelo contexto do desafio, de uma verificada nas quebras de linha, pois a utilização do <br> pode estar ocasionando esses espaçamentos indevidos.

Ex.:
`<p> primeiro codigo </p><br>
<p> segundo codigo </p><br>
<p> terceiro codigo </p><br>
<p> quarto codigo </p><br>`

**Comment by GustavoCarvalhoRodrigues** [comment]
<p><br>*Um<br>**Dois<br>***Três<br>****Quatro</p>
   <p><br>****Quatro<br>***Três<br>**Dois<br>*Um</p>, eu fiz desse jeito.

**Comment by GustavoCarvalhoRodrigues** [comment]
<h1>Desafio das Mensagens</h1>
    <p>* Um</p>
    <p>** Dois</p>
    <p>*** Três</p>
    <p>**** Quatro</p>
    <br>
    <p>**** Quatro</p>
    <p>*** Três</p>
    <p>** Dois</p>
    <p>* Um</p>

**Comment by flameastro** [comment]
@isadhoramauricio Por favor, fecha essa ISSUEE!!
